toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss

```diff
@@ -52,6 +52,22 @@ $author-icon-background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org
 
 		.js & {
 			display: inline-block;
+
+			&.corresponding {
+				margin-right: 20px;
+
+				&:after {
+					content: ' ';
+					display: inline-block;
+					position: absolute;
+					background: $author-icon-background;
```

Comment:
I realise it's not part of this PR but I noticed that this variable is defined in the 50-components file when it should be in the 10-settings file for this